externs/esrijson.js

```diff
@@ -0,0 +1,198 @@
+
+/**
+ * @fileoverview Externs for EsriJSON.
+ * @see http://resources.arcgis.com/en/help/rest/apiref/geometry.html
+ * @externs
+ */
+
+
+
+/**
+ * @constructor
+ */
+var EsriJSONObject = function() {};
+
+
+/**
+ * @type {!EsriJSONCRS}
+ */
+EsriJSONObject.prototype.spatialReference;
+
+
+
+/**
+ * @constructor
+ */
+var EsriJSONCRS = function() {};
+
+
+/**
+ * CRS well know identifier.
+ * @type {!number}
```

Comment:
Primitive types are not nullable by default, so I think that you can safely remove the exclamation mark.